src/views/Playlist/PrivatePlaylist.vue

```diff
@@ -17,10 +17,26 @@ get(Number(props.id))
 </script>
 
 <template>
-  <h1>{{ trackCollection?.name }}</h1>
+  <h2>{{ trackCollection?.name }}</h2>
   <ol class="list-decimal list-inside">
     <li v-for="track in trackCollection.tracks" :key="`track-${track.id}`">
       {{ track.title }}
     </li>
   </ol>
 </template>
+
+<style scoped>
```

Comment:
Could you revert these changes? I don't know if they're in line with the styleguide - if it is, it shouldn't be done this way (scoped, custom styles).

Reply:
Maybe, I don't know much about tailwind, but I wouldn't see it as part of the problem you're creating this PR for.

Reply:
Nothing this time - just saying that this is not a good way of doing it. We'll have a round of design-changes to the project in a later state, where we'll just concentrate on getting the design straight and define a way of how to do it. I'm not the right person to give a review here...